app/src/main/kotlin/Main.kt

```diff
@@ -15,17 +15,7 @@ import androidx.compose.ui.window.application
 fun App() {
     var text by remember { mutableStateOf("Hello, World!") }
 
-    MaterialTheme {
-        Button(onClick = {
-            text = "Hello, Desktop!"
-        }) {
-            Text(text)
-        }
-    }
+    MaterialTheme { Button(onClick = { text = "Hello, Desktop!" }) { Text(text) } }
```

Comment:
Looks ugly...

Reply:
Maybe, but on the other hand ktfmt consistent and produces the same formatting on different inputs.